react/testing_5/src/screens/Dashboard/components/SignupForm/index.test.js

```diff
@@ -0,0 +1,114 @@
+const faker = require('faker');
+const puppeteer = require('puppeteer');
+
+const person = {
+  firstName: faker.name.firstName(),
+  lastName: faker.name.lastName(),
+  email: faker.internet.email()
+};
+
+describe('Signup Form', () => {
+  let browser, page;
+
+  beforeAll(async () => {
+    browser = await puppeteer.launch();
+    page = await browser.newPage();
+  
+    page.emulate({
+      viewport: {
+      width: 500,
+      height: 900
+      },
+      userAgent: ''
+    });
+    
+  });
+  
+  beforeEach(async () => await page.goto('http://localhost:3000/'));
```

Comment:
As a nice to have, I think this URL should be in a constants file or even a `.env`. What do you think ?